using-timescaledb/data-tiering.md

```diff
@@ -0,0 +1,144 @@
+
+Data tiering provides another mechanism, in addition to other TimescaleDB
+capabilities like compression and data retention, to help manage data storage
+costs.
+
+Using multiple tablespaces can also yields I/O performance benefits. With data
+tiering, you can isolate large scans of historical data from the continual
+read/write workload against recent data (in the default tablespace).
+
+## Creating a Tablespace
+
+The [`move_chunk`][api-move-chunk] function requires multiple tablespaces set up in PostgreSQL, so let's
+start with a quick review of how this works.
+
+First, add a storage mount that will serve as a home for your new tablespace. This
+process will differ based on how you are deployed, but your system administrator
+should be able to arrange setting up the mount point. The key here is to provision
+your tablespace with storage that is appropriate for how its resident data will be used.
+
+To create a [tablespace][] in Postgres:
+
+```sql
+CREATE TABLESPACE history
+OWNER postgres
+LOCATION '/mnt/history':
+```
+
+Here we are creating a tablespace called `history` that will be
+owned by the Postgres user and it will use the storage mounted at `/mnt/history`.
+
+Now we are ready to leverage this new tablespace and underlying storage for TimescaleDB
+chunks and their indexes.
+
+## Move Chunks :community_edition: [](move_chunks)
+
+Now that we have set up a new, empty tablespace, we can move individual chunks
+from their default tablespace to this new secondary tablespace.  The move
+chunks command also allows you to move indexs belonging to those chunks to this
+secondary tablespace (or another on).
+
+In addition, the [`move_chunk`][api-move-chunk] functionality provides the
+ability to "reorder" the chunk during the migration in order to enable faster
+queries.  This behavior is similar to [`reorder_chunk`][reorder_chunk]; please
+see that documentation for more information.
+
+To determine which chunks to move, we can list chunks that fit a specific
+criteria.  For example, to identify chunks older than two days:
+
+```sql
+SELECT show_chunks('conditions', older_than => INTERVAL '2 days');
+```
+
+In this example, we will move `_timescaledb_internal._hyper_1_4_chunk` along
+with its index over to `history` using the following command, while reordering
+the chunk based on its time index:
+
+```sql
+SELECT move_chunk(
+  chunk => '_timescaledb_internal._hyper_1_4_chunk',
+  destination_tablespace => 'history',
+  index_destination_tablespace => 'history',
+  reorder_index => '_timescaledb_internal._hyper_1_4_chunk_netdata_time_idx',
+  verbose => TRUE
+);
+```
+Once this successfully executes, we can verify that our chunk now lives on the
+`history` tablespace by querying `pg_tables` to list all of the chunks that
+are on `history`:
+
+```sql
+SELECT tablename from pg_tables
+  WHERE tablespace = 'history' and tablename like '_hyper_%_%_chunk';
+```
+
+As you will see, the target chunk is now listed as residing on `history`; we
```

Comment:
Here and directly below, I feel like there should be example output.

Reply:
Don't disagree, but going to skip for now for time...